crates/nu-command/src/system/run_external.rs

```diff
@@ -269,6 +269,14 @@ impl ExternalCommand {
                                         "'{}' was not found, did you mean '{s}'?",
                                         self.name.item
                                     )
+                                } else if self.name.item == s {
+                                    let sugg = engine_state.which_module_has_decl(s.as_bytes());
+                                    if let Some(sugg) = sugg {
+                                        let sugg = String::from_utf8_lossy(sugg);
+                                        format!("did you mean '{sugg} {s}'?")
+                                    } else {
+                                        format!("did you mean '<module> {s}'?")
```

Comment:
I'm thinking this might not be desired: If the command is not in a module, the suggestion should be just the default "did you mean", IMO.